src/user-management/user-details/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Management</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="constants.js" defer></script>
+    <script src="utils.js" defer></script>
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1>User Management</h1>
+    </header>
+    <section id="main-section">
+      <div class="user-details">
+        <div class="user-details__personal"></div>
+        <ul class="user-details__accordion hide accordion">
+          <li class="accordion__academic">
+            <div class="visible-content">
+              <h2>Academics or Professional Details</h2>
```

Comment:
can't we just keep it as `details` or `user details`

Reply:
Hi @RitikJaiswal75 as per the design this was supposed to be called `Academics or Professional Details`.

Reply:
doesn't it sound too big as it is a heading for a dropdown, not a page heading, if it could be small it would look better.